client/src/components/FormAction/FormAction.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import SilverStripeComponent from 'lib/SilverStripeComponent';
 import castStringToElement from 'lib/castStringToElement';
+import cx from 'classnames';
```

Comment:
This is, or at least _was_ a convention back when it was part of the main React repo. I think it's just force of habit.